packages/drizzle/src/postgres/schema/buildDrizzleTable.ts

```diff
@@ -81,6 +82,13 @@ export const buildDrizzleTable = ({
         break
       }
 
+      case 'vector': {
+        const builder = vector(column.name, { dimensions: column.dimensions })
```

Comment:
Reviewing on mobile, are there any other properties for the options arg besides `dimensions` that we should pass through?

Reply:
Looks like that is the only thing drizzle supports on the config object for PgVector 👍